src/DynamoCoreWpf/Controls/AddonsTreeView.xaml.cs

```diff
@@ -13,6 +13,10 @@ public partial class AddonsTreeView : UserControl
         public AddonsTreeView()
         {
             InitializeComponent();
+
+            // In future DataContext will be set SearchViewModel in binding, but for now set it to null. 
+            // Therefore we can escape errors in VS Output.
+            this.DataContext = null;
```

Comment:
Perhaps the better way of putting this would be:

```csharp
// Invalidate the DataContext here because it will be set at a later 
// time through data binding expression. This way debugger will not 
// display warnings for missing properties.
```